docs/openshift.asciidoc

```diff
@@ -0,0 +1,140 @@
+[id="{p}-openshift"]
+== Openshift
+
+This section will help you run ECK on Openshift
+
+NOTE: Only Elasticsearch and Kibana are supported on Openshift or with a secured cluster for the moment.
```

Comment:
Can you clarify what this means? The "or with a secured cluster" doesn't have a clear meaning to me at least

Reply:
You are right. I have removed it because it is more misleading than helpful. K8S clusters can be secured with PSP. Like the `restricted` SCC on Openshift, restricted PSP may prevent the deployment of the APM server _(which is not compatible with some of these policies)_